Migration/jcl/groovy/JCLtoDBB.groovy

```diff
@@ -573,7 +582,8 @@ def processAllocOption( parm )
 					}
 					if (allocations.size() > 2 && allocations[2].length() > 0)
 					{
-						options << "directory(${allocations[2]})"
+                        //* updated to use dir instead of directory for BPXWDYN utility
```

Comment:
I think we can get rid of the comment.

Reply:
Updated the changes and created a new pull request - https://github.com/IBM/dbb/pull/126.  This pull request seems to be from my personal github account.

Reply:
Hi @dennis-behm , Updated the changes and created a new pull request - https://github.com/IBM/dbb/pull/126. This pull request seems to be from my personal github account.